package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react-router-dom": "^5.2.0"
+  }
```

Comment:
I think this package.json was adde by mistake. Please delete this file.

Reply:
File was removed in the latest commit on my branch.